wp-parsely.php

```diff
@@ -21,6 +21,189 @@
 	'3.1',
 ];
 
+/**
+ * Class to hold Parse.ly loading info
+ * Will prevent this information from being queried multiple times
+ * by keeping the status in a static variable
+ */
+final class Parsely_Loader_Info {
+
+	/**
+	 * Strings for Parse.ly integration types.
+	 */
+	const INTEGRATION_TYPE_MUPLUGINS        = 'MUPLUGINS';
+	const INTEGRATION_TYPE_MUPLUGINS_SILENT = 'MUPLUGINS_SILENT';
+	const INTEGRATION_TYPE_SELF_MANAGED     = 'SELF_MANAGED';
+	const INTEGRATION_TYPE_NONE             = 'NONE';
+
+	/**
+	 * String for when no wp-parsely version can be detected.
+	 */
+	const VERSION_NONE = 'NONE';
+
+	/**
+	 * Strings for Parse.ly service types.
+	 */
+	const SERVICE_TYPE_PAID = 'PAID';
+	const SERVICE_TYPE_NONE = 'NONE';
+
+	/**
+	 * @var boolean
+	 */
+	private static $active;
+
+	/**
+	 * @var string
+	 */
+	private static $integration_type;
+
+	/**
+	 * @var array The Parse.ly WordPress options dictionary.
+	 */
+	private static $parsely;
```

Comment:
For clarity, I would suggest to rename this variable and related functions to `*parsely_option` or similar

Reply:
You are right. This should be renamed. I have refactored this property, as well as its getter/setter methods, in the latest commit.